editor/components/block-list/insertion-point.js

```diff
@@ -21,7 +21,7 @@ class BlockInsertionPoint extends Component {
 
 		this.onBlurInserter = this.onBlurInserter.bind( this );
 		this.onFocusInserter = this.onFocusInserter.bind( this );
-		this.onClick = this.onClick.bind( this );
+		this.onMouseDown = this.onMouseDown.bind( this );
```

Comment:
My immediate thought is: how does this work with a keyboard user? I forget if `mouseDown` is fired for keyboard events, but I feel like it wouldn't be. 🤷‍♂️

Reply:
Oh yes, interesting. This doesn't work with a keyboard, but click does. (ノ°Д°)ノ︵ ┻━┻

So, in lieu of that — do you have any other ideas on how to address this? I.e. is there a way to prevent the onClick on the sibling inserter to propogate into a mouseDown event also?

I also have a plan B idea, but worth exploring the above first it seems.

Reply:
I'm not sure what to do to fix this exactly, but I know changing the `render()` method in `editor/components/block-list/insertion-point.js` to:

```js
// editor/components/block-list/insertion-point.js
render() {
	const { isInserterFocused } = this.state;
	const { showInsertionPoint, showInserter } = this.props;

	return (
		<div className="editor-block-list__insertion-point">
			{ true && <div className="editor-block-list__insertion-point-indicator" /> }
			{ true && (
				<div className={ classnames( 'editor-block-list__insertion-point-inserter', { 'is-visible': true } ) }>
					<IconButton
						icon="insert"
						className="editor-block-list__insertion-point-button"
						onClick={ this.onClick }
						label={ __( 'Insert block' ) }
						onFocus={ this.onFocusInserter }
						onBlur={ this.onBlurInserter }
					/>
				</div>
			) }
		</div>
	);
}
```

fixes it. I'll look through the event propagation chain but it's quite a maze! 😃